strax/run_selection.py

```diff
@@ -283,6 +283,66 @@ def define_run(self: strax.Context,
                            " run definition")
 
 
+@strax.Context.add_method
+def available_for_run(self: strax.Context,
+                      run_id: str,
+                      include_targets: ty.Union[None, list, tuple, str] = None,
+                      exclude_targets: ty.Union[None, list, tuple, str] = None,
+                      pattern_type: str = 'fnmatch') -> pd.DataFrame:
+    """
+    For a given single run, check all the targets if they are stored.
+        Excludes the target if never stored anyway.
+    :param run_id: requested run
+    :param include_targets: targets to include e.g. raw_records,
+        raw_records* or *_nv. If multiple targets (e.g. a list) is
+        provided, the target should match any of the arguments!
+    :param exclude_targets: targets to exclude e.g. raw_records,
+        raw_records* or *_nv. If multiple targets (e.g. a list) is
+        provided, the target should match none of the arguments!
+    :param pattern_type: either 'fnmatch' or 're'
```

Comment:
What does `fnmatch` or `re` stand for?

Reply:
Proper python constructs 😉 :
- https://docs.python.org/3/library/fnmatch.html
- https://docs.python.org/3/library/re.html

Reply:
added a sentence or two